cmd/osbuild-worker-executor/handler_result.go

```diff
@@ -38,7 +38,7 @@ func handleResult(logger *logrus.Logger, config *Config) http.Handler {
 				return
 			}
 
-			fss := http.FileServer(http.Dir(filepath.Join(config.BuildDirBase, "build/output")))
```

Comment:
Same comment as above, maybe a very short coment why we do not serve output only but everything